### ISSUE TITLE
Event with changed data on import

### ISSUE THREAD

**Issue by ascheider** [BlackbitDigitalCommerce/pimcore-data-director#83]
Is there any possibility to get data which was changed on import?
Like fieldname and data?
Currently we need do load previous version and compare the data, to get delta.
It would be really helpful to get the data directly from import and process them for third party.

**Comment by BlackbitDevs** [comment]
From version 3.3 the changes get written to the version note.
I could also provide this directly. Currently I am working on some memory problems and storing even more data of the imported objects would worsen this problem. But I put this onto my todo list...

**Comment by ascheider** [comment]
@BlackbitDevs Yes in postUpdate event is it perfect and would be very nice, it will reduce so much workload to get new delta data.
